src/mapml/layers/TemplatedFeaturesLayer.js

```diff
@@ -76,14 +80,41 @@ export var TemplatedFeaturesLayer =  L.Layer.extend({
         this._onMoveEnd();
     },
 
+    _removeCSS: function () {
+        let toDelete = this._container.querySelectorAll("link[rel=stylesheet],style");
+        for(let i = 0; i < toDelete.length;i++){
+            let parent = toDelete[i].parentNode;
+            parent.removeChild(toDelete[i]);
+        }
+    },
+
     _onMoveEnd: function() {
+      let history = this._map.options.mapEl._history;
+      let current = history[history.length - 1];
+      let previous = history[history.length - 2];
+      if(!previous) previous = current;
```

Comment:
Good opportunity for [nullish coalescing](https://twitter.com/thelittlestdev/status/1415213119731814401?s=20&t=eoaqarkZSLddrw1w62iG-g): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator